server/start.go

```diff
@@ -214,6 +217,20 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 	home := cfg.RootDir
 	var cpuProfileCleanup func()
 
+	if sdkPprofAddr := ctx.Viper.GetString(flagSDKPprofAddr); sdkPprofAddr != "" {
+		go func() {
+			if sdkPprofAddr == "default" {
+				sdkPprofAddr = "localhost:6061"
+			}
```

Comment:
Wait should we make default be off?

Seems like a high penalty to make all nodes pay by default

Reply:
It should be off by default if the user does not provide the flag.

The flag value of "default" string yields localhost:6061.

Otherwise, it uses the provided value for the address

Reply:
ohh

Reply:
it defaults to `""` (https://github.com/osmosis-labs/cosmos-sdk/pull/387/files#diff-97289af2ab66016f168df6fccac70626d125f170e5cf0841fcdc8fa624999608R151)